### PR TITLE
OAP-18 Create basic framework to view books through OAPEN (rough is OK)

### PR DESCRIPTION
**Pull request by justinoboyle**

### CODE REVIEW

**Comment by justinoboyle** [comment]
this closes OAP-11

File: web/lib/oapen/OAPENTypes.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+export type OAPENTypeID = string;
+export type OAPENHandle = string;
+export type OAPENType = string;
+export type OAPENExpandable = string;
+export type OAPENBitstreams = string;
+export type OAPENApiRelativePath = string;
+export type URL = string;
+
+export interface Expandable {
```
Comment (celinanperalta) [inline web/lib/oapen/OAPENTypes.ts:9]: Comment on what this means please

File: web/lib/oapen/items.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+export const getItemSingleMetadata = async (
+  uuid: string
+): Promise<RawMetadataItem[]> => {
+  const rawText = await get(
+    ["items", uuid, "metadata"].map(encodeURIComponent).join("/")
+  );
+  return rawText as RawMetadataItem[];
+};
+
+export const combineItemMetadata = (
```
Comment (celinanperalta) [inline web/lib/oapen/items.ts:56]: nit: necessary to export?

**Review by celinanperalta — APPROVED** [review]

File: web/lib/oapen/items.ts (hunk 2 above)
Comment (justinoboyle) [reply web/lib/oapen/items.ts:56]: not for now, nice catch

File: web/lib/oapen/OAPENTypes.ts (hunk 1 above)
Comment (justinoboyle) [reply web/lib/oapen/OAPENTypes.ts:9]: added:
```
  // OAPEN returns a list of "expandables" -- things you can further query for additionally (like metadata). Once the expandable is added to the object, it's removed from this list.
```

**Pull request merged**